config/rbac/role.yaml

```diff
@@ -104,20 +104,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - memcached.openstack.org
-  resources:
-  - memcacheds/finalizers
-  verbs:
-  - update
-- apiGroups:
-  - memcached.openstack.org
-  resources:
-  - memcacheds/status
```

Comment:
I think this _should_ be ok. But we do create our own instance of memcached, so we might need to keep this change in mind if we run into any issues related to checking the status of our memcached instance. They all seem to be set by the memcached controller atm though:
```
❯ oc get memcacheds horizon-memcached -o jsonpath='{.status}'
{"conditions":[{"lastTransitionTime":"2023-04-06T00:25:00Z","message":"Setup complete","reason":"Ready","status":"True","type":"Ready"},{"lastTransitionTime":"2023-04-06T00:25:00Z","message":"Deployment completed","reason":"Ready","status":"True","type":"DeploymentReady"},{"lastTransitionTime":"2023-04-06T00:24:49Z","message":"Exposing service completed","reason":"Ready","status":"True","type":"ExposeServiceReady"},{"lastTransitionTime":"2023-04-06T00:24:49Z","message":"Service config create completed","reason":"Ready","status":"True","type":"ServiceConfigReady"}]}%
```